### ISSUE TITLE
non-ascii response headers in fetch result in panic

### ISSUE THREAD

**Issue by nurmohammed840** [denoland/deno#8576]
I got this error...
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ToStrError { _priv: () }', D:\a\deno\deno\op_crates\fetch\lib.rs:159:53
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
There is no stack trace, Just this message.

**Comment by lucacasonato** [comment]
What is the code you are trying to run?

**Comment by bartlomieju** [comment]
Looks like a response header that can't be turned into a string

**Comment by lucacasonato** [comment]
Looks like the fetch spec mandates that unparsable headers be ignored: https://fetch.spec.whatwg.org/#concept-header-list-get-structured-header

**Comment by nurmohammed840** [comment]
> What is the code you are trying to run?

I don't know what kind of code create this issue. There is no more error information I have. Just only this error message

**Comment by nurmohammed840** [comment]
But still throw same Error. ( deno --version  : 1.6.0 )
```ts
// This program throw.Even handling `catch` block
fetch("https://www.laptoparena.net/lg/notebook-lg-gram-15.6”-15z90n-n.aps9u1-silver-48713").catch(() => console.log("Error"))
```

**Comment by bartlomieju** [comment]
@nurmohammed840 the fix is not a part of `1.6.0`. It will be released in `1.6.1` in a few days.
